src/core/services/report_service.py

```diff
@@ -105,12 +105,24 @@ async def __notify_member_about_finished_shift(self, member: Member, bot: Applic
             and not await self.__member_repository.is_unreviewed_report_exists(member.id)
         ):
             await self.__finish_shift_with_all_reports_reviewed(member.shift)
+            lombaryers_case = ''
+            numbers_lombaryers = member.numbers_lombaryers
+            if numbers_lombaryers == 1 or (numbers_lombaryers > 20 and (numbers_lombaryers % 10) == 1) and (
+                    numbers_lombaryers % 100) != 11:
+                lombaryers_case = 'ломбарьерчик'
+            elif (1 < numbers_lombaryers < 5) or (numbers_lombaryers > 20 and 1 < (numbers_lombaryers % 10) < 5):
+                lombaryers_case = 'ломбарьерчика'
+            elif numbers_lombaryers == 0 or (1 < numbers_lombaryers < 20) or (numbers_lombaryers % 10) == 0 or (
+                    numbers_lombaryers % 100) >= 11 or (numbers_lombaryers % 10) >= 5 or (
+                    numbers_lombaryers % 100) >= 10:
+                lombaryers_case = 'ломбарьерчиков'

```

Comment:
Есть ошибка: 112 ломбарьерчика.
И можно сделать значительно проще.
```suggestion
    PLURAL = 'ломбарьерчиков'
    SINGULAR = 'ломбарьерчик'
    GENITIVE = 'ломбарьерчика'
    last_two_digits = numbers_lombaryers % 100
    if 11 <= last_two_digits <= 19:
        return PLURAL
    last_digit = last_two_digits % 10
    if last_digit == 1:
        return SINGULAR
    if 1 < last_digit < 5:
        return GENITIVE
    return PLURAL

```